Formula/chapel.rb

```diff
@@ -23,19 +23,16 @@ def install
     ENV["CHPL_HOME"] = libexec
     # This is for mason
     ENV["CHPL_RE2"] = "bundled"
+    ENV["CHPL_GMP"] = "system"
+    ENV["CHPL_INCLUDE_PATH"] = "/usr/local/include"
+    ENV["CHPL_LIB_PATH"] = "/usr/local/lib"
```

Comment:
```suggestion
    ENV["CHPL_INCLUDE_PATH"] = HOMEBREW_PREFIX/"include"
    ENV["CHPL_LIB_PATH"] = HOMEBREW_PREFIX/"lib"
```
Similarly for the test block.